### PR TITLE
Add CONTRIBUTE.md to help new contributors.

### PR DESCRIPTION
**Pull request by m-kuhn**
This file will be shown when people make new pull requests.

contribute
See here:
https://github.com/blog/1184-contributing-guidelines#contributors-wanted-inquire-within

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    ln -s scripts/prepare-commit.sh .git/hooks/pre-commit
+
+Getting your pull request merged
+--------------------------------
+
+This is a volunteer project, so sometimes it may take a little while to merge
+your pull request.
+
+There is a [guide with hints for getting your pull requests merged](https://github.com/m-kuhn/QGIS-Website/blob/qtcreator.rst/source/site/getinvolved/development/git.rst#pull-requests)
```
Comment (alexbruy) [inline CONTRIBUTE.md]: Maybe this link should point to the official repo?

Comment (m-kuhn) [inline CONTRIBUTE.md]: Actually it should point to the developer guide on the website but the PR is not yet ready to be merged, so in the meantime I thought I'd point to this.
https://github.com/qgis/QGIS-Website/pull/297

**Comment by nirvn** [comment]
@m-kuhn , the pre-commit hook is a very useful tip.

**Comment by m-kuhn** [comment]
Say thank you to @jef-n

**Comment by jef-n** [comment]
and @manisandro

**Comment by nirvn** [comment]
Thank you @jef-n and @manisandro ;)

**Pull request merged**